src/config/musl.md

```diff
@@ -0,0 +1,99 @@
+# musl
+
+[musl](https://musl.libc.org/) is a libc implementation which strives to be
+lightweight, fast, simple, and correct.
+
+Void officially supports musl by using it in its codebase for all target
+platforms except i686. Additionally, all the packages in our official
+repositories are available with musl-linked binaries in addition to their glibc
+counterparts.
+
+Currently, there are nonfree and debug subrepositories for musl, but no multilib
+subrepo.
+
+Generally speaking, a musl based system is a good choice for servers, embedded
+solutions and old PCs, but it may not be worth the effort to use on a modern
+desktop computer.
+
+## Possible problems
+
+Some programs (mostly graphical applications) will work incorrectly, or
+segfault, when run under musl. This may be due to programs expecting some
+glibc-specific behavior. However, currently most breakages are caused by default
+stack size differences — if you are a software developer, you can try
+rebuilding musl with a bigger stack size and see if that helps.
```

Comment:
This is not necessary, you can use a shell to launch your program and run `ulimit -s hard` on said shell. This completely avoids the issues with limited stack size on musl. We could include a short script that launches something like qutebrowser (which crashes due to the stack issue) through bash.

A script like this:

```shell
#!/usr/bin/env bash

ulimit -s hard
"$@"
```

Reply:
Ah okay, thanks for pointing this out - i've update the PR accordingly.

Reply:
@ericonr this is wrong, the musl stack size limitation is thread stack size, not main stack size, and that is *not* influenced by process limits. The main stack size tends to be 8M or so regardless of the libc you run.

Reply:
@q66: So should i put the original text back in?

Reply:
no, you already removed the paragraph

Reply:
Okay, thanks.

Reply:
@q66 sorry for the wrong information. Using ulimit seemed to have solved one issue I had, which was opening any reddit page on qutebrowser while logged into reddit, causing a seg-fault. I'm not sure what led me to believe it had fixed the issue, but testing it now showed me the issue is still present.Weird, but ok.